rust/src/domains/mod.rs

```diff
@@ -8,6 +8,7 @@
 //! Each metric (see [`crate::metrics`]) is associated with certain data domains. 
 //! The [`Domain`] trait is implemented for all domains used in OpenDP.
 
+#[cfg(feature="ffi")]
```

Comment:
This is used by `DataDomain` outside of ffi.

Reply:
Thanks, good catch! Per discussion, I'll change this to `feature="ffi"`.